### PR TITLE
[TASK_3]Implement endpoints to get products information

### PR DESCRIPTION
**Pull request by artem-blazhko**
Basic evaluation criterias are met.

All additional tasks are done:
- async functions are used (+1)
- ES6 modules are used  (+1)
- Webpack is configured (+1)
- SWAGGER documentation is created (+1)
- Lambda handlers are covered by UNIT tests (+1)
- getProductsList, getProductsById are written in separated files (+1)
- main error scenarios are handled (+1)

[Link to application](https://d1cw0fkrv9p4ak.cloudfront.net/)
------------------------------
[Link to front-end PR](https://github.com/artem-blazhko/nodejs-aws-fe/pull/2)
------------------------------
[Link to get all products](https://n7da350mok.execute-api.eu-west-1.amazonaws.com/dev/products)
------------------------------
image

[Link to get a product by id](https://n7da350mok.execute-api.eu-west-1.amazonaws.com/dev/products/5)
------------------------------
image

[Link to get a product which doesn't exist](https://n7da350mok.execute-api.eu-west-1.amazonaws.com/dev/products/12)
------------------------------
image

Coverage by tests
------------------------------
image

### CODE REVIEW

**Comment by MaxPlav** [comment]
please handle an error when not a numeric id is provided https://n7da350mok.execute-api.eu-west-1.amazonaws.com/dev/products/asdasd. It should be not 404, but 400 and another error message.

File: product-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#    - exclude-me.js
+#    - exclude-me-dir/**
+
+functions:
+  getProductsList:
+    handler: handler.getProductsList
+    events:
+      - http:
+          path: products
+          method: get
```
Comment (MaxPlav) [inline product-service/serverless.yml:74]: "cors: true" should be set for each endpoint, to bypass CORS policy

File: product-service/swagger.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    "/products/{productId}": {
+      "get": {
+        "description": "Returns a specific product requested by productId",
+        "operationId": "getProductById",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
```
Comment (MaxPlav) [inline product-service/swagger.json:58]: should be "name": "productId"

File: product-service/swagger.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        },
+        "price": {
+          "type": "number",
+          "example": 102
+        },
+        "title": {
+          "type": "string",
+          "example": "Title"
+        },
+        "description": {
```
Comment (MaxPlav) [inline product-service/swagger.json:104]: what about "image" field?

File: product-service/swagger.json [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    "description": "Product service",
+    "contact": {
+      "name": "Artem Blazhko"
+    },
+    "license": {
+      "name": "MIT"
+    }
+  },
+  "host": "AWS",
+  "basePath": "/test",
```
Comment (MaxPlav) [inline product-service/swagger.json:15]: please fix the "basePath"

File: product-service/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "git+https://github.com/artem-blazhko/aws-shop-be.git"
+  },
+  "author": "Artem Blazhko",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/artem-blazhko/aws-shop-be/issues"
+  },
+  "homepage": "https://github.com/artem-blazhko/aws-shop-be#readme",
+  "exports": "./handler.js",
+  "scripts": {
```
Comment (wwrcth) [inline product-service/package.json:17]: Why don't you want to add a command for deploy? Would be very useful :)

File: product-service/serverless.yml (hunk 1 above)
Comment (artem-blazhko) [reply product-service/serverless.yml:74]: I have appropriate headers instead

**Pull request merged**